Description: Fixed logPolar
  just copied the new line from upstream
Author: Alf Gaida <agaida@siduction.org>
Last-Update: 2020-02-08

--- nomacs-3.12.0+dfsg.orig/src/DkCore/DkImageStorage.cpp
+++ nomacs-3.12.0+dfsg/src/DkCore/DkImageStorage.cpp
@@ -1424,7 +1424,7 @@ void DkImage::mapGammaTable(cv::Mat& img
 	qDebug() << "gamma computation takes: " << dt;
 }
 
-void DkImage::logPolar(const cv::Mat& src, cv::Mat& dst, CvPoint2D32f center, double scaleLog, double angle, double scale) {
+void DkImage::logPolar(const cv::Mat& src, cv::Mat& dst, cv::Point2d center, double scaleLog, double angle, double scale) {
 
 	cv::Mat mapx, mapy;
 

--- nomacs-3.12.0+dfsg.orig/src/DkCore/DkImageStorage.h
+++ nomacs-3.12.0+dfsg/src/DkCore/DkImageStorage.h
@@ -95,7 +95,7 @@ public:
 	static void mapGammaTable(cv::Mat& img, const QVector<unsigned short>& gammaTable);
 	static void gammaToLinear(cv::Mat& img);
 	static void linearToGamma(cv::Mat& img);
-	static void logPolar(const cv::Mat& src, cv::Mat& dst, CvPoint2D32f center, double scaleLog, double angle, double scale = 1.0);
+	static void logPolar(const cv::Mat& src, cv::Mat& dst, cv::Point2d center, double scaleLog, double angle, double scale = 1.0);
 	static void tinyPlanet(QImage& img, double scaleLog, double angle, QSize s, bool invert = false);
 #endif
